.github/workflows/pull-request.yml

```
@@ -9,482 +9,475 @@
+          uses: tj-actions/changed-files@v45
+          with:
+            files: |
+              .env*
+              vitest.config.js
+              src/App.tsx
+              .github/**
+              env.example
+              .node-version
+              .husky/**
+              scripts/**
+              src/style/** 
```

YAMLlint (1.35.1)

[error] trailing spaces

(trailing-spaces)